src/main/java/TaskManager.java

```diff
@@ -66,27 +67,29 @@ public void addTask(String option, String taskDescription){
         description = description.trim();
         String time = taskDescription.substring(sepIndex + 3);
         time = time.trim();
-        if(option.equals("deadline")){
+        if(option.equals("deadline")) {
             tasks[taskCount++] = new Deadline(description, time);
-        }
-        else if(option.equals("event")){
+        } else if(option.equals("event")) {
             tasks[taskCount++] = new Event(description, time);
         }
     }
 
     public void listTasks(){
         System.out.println("\t" + "-".repeat(60));
         System.out.println("\t Here are the tasks in your list:");
-        for(int i = 0;i < taskCount; i++){
-            System.out.println("\t\t " + (i + 1) + "." + tasks[i].toString());
-        }
-        if(taskCount == 0){
+        if(taskCount == 0) {
             System.out.println("\t No task recorded.");
+            System.out.println("\t" + "-".repeat(60));
+            return;
+        }
+
+        for(int i = 0;i < taskCount; i++) {
```

Comment:
Great job on using curly braces for your for loop eventhough there's only 1 line of code